test/systems/plonk/placeholder/performance.cpp

```diff
@@ -113,25 +113,18 @@ class placeholder_performance_test_base {
         return step_list;
     }
 
-    template<typename fri_type, typename FieldType>
-    static typename fri_type::params_type create_fri_params(std::size_t degree_log, const int max_step = 1) {
-        typename fri_type::params_type params;
-        math::polynomial<typename FieldType::value_type> q = {0, 0, 1};
-
-        constexpr std::size_t expand_factor = 7;
-
-        std::size_t r = degree_log - 1;
-
-        std::vector<std::shared_ptr<math::evaluation_domain<FieldType>>> domain_set =
-            math::calculate_domain_set<FieldType>(degree_log + expand_factor, r).get();
-
-        params.r = r;
-        params.D = domain_set;
-        params.max_degree = (1 << degree_log) - 1;
-        params.step_list = generate_random_step_list(r, max_step);
+    typename fri_type::params_type create_fri_params(
+        std::size_t degree_log, const int max_step = 1, std::size_t expand_factor = 7) {
+    std::size_t r = degree_log - 1;
+
+    return typename fri_type::params_type(
+        (1 << degree_log) - 1, // max_degree
+        math::calculate_domain_set<FieldType>(degree_log + expand_factor, r),
+        generate_random_step_list(r, max_step),
+        expand_factor
+    );
```

Comment:
I've seen it so many times by now. Think, we should consider moving it to shared file

Reply:
Yes, we'll probably make this a constructor to the fri_type::params_type class sometime later...